_data/hosting.yml

```
@@ -380,9 +380,10 @@ websites:
 
   - name: Netlify
     url: https://www.netlify.com
-    twitter: Netlify
-    facebook: netlify
     img: netlify.png
+    tfa:
+      - totp
+    doc: https://url.netlify.com/rk5a6rzJv
```

The official documentation is probably more suited than a blog entry.
```suggestion
    doc: https://docs.netlify.com/accounts-and-billing/user-settings/#two-factor-authentication-2fa
```